cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/type/impl/CassandraCalendarDateType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.cassandra.type.impl;
+
+import org.hibernate.MappingException;
+import org.hibernate.engine.spi.Mapping;
+import org.hibernate.ogm.type.impl.AbstractGenericBasicType;
+import org.hibernate.type.descriptor.java.CalendarDateTypeDescriptor;
+
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Cassandra doesn't do Calendar, but it does do Date, so convert them.
```

Comment:
What happens to the TZ information of calendars? As per [this doc](http://docs.datastax.com/en/cql/3.0/cql/cql_reference/timestamp_type_r.html), Cassandra's time stamps support storing a TZ offset. I guess that'd require to put forward calendars as formatted strings rather than `Date`.